### PR TITLE
[API-691] Clean up documentation for gears, routes, running races, and segment efforts endpoints

### PR DESCRIPTION
**Pull request by hariniriyer**
#### Changes

Added the sample response for Gears Get Equipment.
Added the sample response for Segment Efforts, GetSegmentEffortsById
Added the sample response for Segment Efforts, exploreSegments

### CODE REVIEW

File: swagger/swagger.json.mustache [hunk 1]
```diff
@@ -1239,6 +1416,18 @@
...
+            "examples": {
+                "application/json": {
+                    "id": "b1231",
+                    "primary": false,
+                    "resource_state": 3,
+                    "distance": 388206,
+                    "brand_name": null,
+                    "model_name": null,
+                    "frame_type": 3,
+                    "description": ""
```
Comment (MateoAO) [inline swagger/swagger.json.mustache]: Data here would be nice

File: swagger/swagger.json.mustache [hunk 2]
```diff
@@ -1239,6 +1416,18 @@
...
             "schema": {
               "$ref": "{{reference_prefix}}/gear.json#/DetailedGear"
+            },
+            "examples": {
+                "application/json": {
+                    "id": "b1231",
+                    "primary": false,
+                    "resource_state": 3,
+                    "distance": 388206,
+                    "brand_name": null,
```
Comment (MateoAO) [inline swagger/swagger.json.mustache]: Sample brand & model would be good

File: swagger/swagger.json.mustache [hunk 3]
```diff
@@ -388,6 +388,123 @@
...
+                            "resource_state": 2,
+                            "name": "Segment One",
+                            "activity_type": "Ride",
+                            "distance": 6297.46,
+                            "average_grade": 4.8,
+                            "maximum_grade": 16.3,
+                            "elevation_high": 416,
+                            "elevation_low": 104.6,
+                            "start_latlng": [
+                                52.98501000581467,
```
Comment (MateoAO) [inline swagger/swagger.json.mustache]: Safe to assume these latlngs have either been scrubbed or are from a public segment? It's kinda fun to pick a well known segment, as it helps people understand. For example https://www.strava.com/segments/4782522 is globally known.

Comment (MateoAO) [reply swagger/swagger.json.mustache]: Also, using scrubbed data for related effort types is fun

Comment (hariniriyer) [reply swagger/swagger.json.mustache]: I have pretty much scrubbed all the location data, names etc. As you mentioned, I will use the well known public segment in which case I dont need to scrub the data that is returned?

**Comment by zackisaacs** [comment]
@hariniriyer I noticed that you pushed up another commit -- are you ready for another round of review? If not, no worries. If so, please ping me with a "@zackisaacs PTAL." I think many folks filter out the Github emails for when a new commit is pushed and only take another look when you explicitly ask for it.

**Comment by hariniriyer** [comment]
@zackisaacs PTAL.

File: swagger/swagger.json.mustache [hunk 4]
```diff
@@ -388,6 +388,123 @@
...
+                    {
+                         "id": 3532004,
+                         "resource_state": 2,
+                         "name": "Alpe d'Huez",
+                         "activity": {
+                         	"id": 1987002,
+                            "resource_state": 1
+                         },
+                         "athlete": {
+                         	"id": 750000,
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: This is a real athlete ID -- let's change it to a big number.

File: swagger/swagger.json.mustache [hunk 5]
```diff
@@ -388,6 +388,123 @@
...
               }
+            },
+            "examples": {
+                "application/json": [
+                    {
+                         "id": 3532004,
+                         "resource_state": 2,
+                         "name": "Alpe d'Huez",
+                         "activity": {
+                         	"id": 1987002,
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: This is a real activity ID -- let's change it to a really big number like `123456789` that isn't and won't be associated with an activity for quite a while.

File: swagger/swagger.json.mustache [hunk 6]
```diff
@@ -388,6 +388,123 @@
...
+                            "state": "RA",
+                            "country": "France",
+                            "private": false,
+                            "hazardous": false,
+                            "starred": false
+                         },
+                         "kom_rank": null,
+                         "pr_rank": null,
+                         "achievements": []
+                    },
```
Comment (zackisaacs) [inline swagger/swagger.json.mustache]: For the sake of keeping the swagger file and our docs a reasonable size, let's get rid of this second segment effort and end after the first one. Opening/closing with `[` and `]` should let folks know that we're returning an array.

**Comment by hariniriyer** [comment]
@zackisaacs I stubbed out those IDs even more and deleted the second segment from the array. PTAL.

**Pull request merged**